e2e_tests/e2e_test.py

```diff
@@ -9,17 +8,22 @@
 from selenium.webdriver.common.keys import Keys
 import time
 from dotenv import load_dotenv
+import os
+
+SELENIUM_DATABASE = "SeleniumTest"
 
 @pytest.fixture(scope='module')
 def test_app():
-    file_path = find_dotenv('.env.test')
-    load_dotenv(file_path, override=True)
-    # Create the new board & update the board id environment variable
-        # Use our test config instead of the 'real' version
-    mongo_db = MongoWrapper().create_database("SeleniumTest")
-    os.environ['DEFAULT_DATABASE'] = "SeleniumTest"
+    try:
+        file_path = find_dotenv('.env.test')
+        load_dotenv(file_path, override=True)

```

Comment:
I don't think I follow - why do you e2e tests load `.env.test`?

Reply:
When running locally I use the `.env.test` file. Hence this check.

Reply:
I don't follow. I expect running e2e tests with .env.test to fail. Unless you mean you temporarily put valid database information in .env.test?

Why not just load .env here?